secant/Features/BackupFlow/RecoveryPhraseDisplayStore.swift

```diff
@@ -15,9 +15,31 @@ enum RecoveryPhraseError: Error {
 }
 
 struct BackupPhraseEnvironment {
-    var mainQueue: AnySchedulerOf<DispatchQueue>
-    var newPhrase: () -> Effect<RecoveryPhrase, RecoveryPhraseError>
-    var pasteboard: UIPasteboard = .general
+    let mainQueue: AnySchedulerOf<DispatchQueue>
+    let newPhrase: () -> Effect<RecoveryPhrase, RecoveryPhraseError>
+    let setPasteboard: (String) -> Void
```

Comment:
I'm trying to wrap my head around how it would be to add this to multiple screens. 

if I understand correctly I would have to add these set/get behavior to each one of the environments with something like this


````Swift
    setPasteboard: { UIPasteboard.general.string = $0 },
    getPasteboard: { UIPasteboard.general.string }
````

Would it make sense to use a protocol default implementation for this?

````Swift
protocol Pasteboardable { //Name sucks but you get the idea
    func setPasteboard(_ string: String)
    var pasteboard: String? // this could be a function as well
}

extension Pasteboardable {
    func setPasteboard(_ string: String) {
        UIPasteboard.general.string = string
    }
    var pasteboard: String? {
        UIPasteboard.general.string
    }
}
````

This way environments benefit from the default implementation. Also Reducers that knowingly deal with copy-paste actions can require an environment that is `Pasteboardable` explicitly. 

The test code won't benefit at all from this since the Reducers and even store might be mockups.

Reply:
I don't think it would be beneficial overall to have the Environment adhere to a protocol(s), but if we want to group these functions together and make our own pseudo dependency we can certainly do that.

Reply:
We can remove the getPasteboard function since it's only used for testing too.  That way it's just a single function.  But either way if the preference is to wrap that we certainly can.